test/goldens/comments.dart.golden

```diff
@@ -203,6 +203,21 @@
 >
 >/**
 #^^^ comment.block.documentation.dart
+> *   /**
+#^^^^^^^^ comment.block.documentation.dart
+> *    * bbb
+#^^^^^^ comment.block.documentation.dart
+#      ^^^^^ comment.block.documentation.dart variable.other.source.dart
+> *    */
+#^^^^^^^^ comment.block.documentation.dart
+> */
+#^^^ comment.block.documentation.dart
+>var d2;
+#^^^ storage.type.primitive.dart
+#      ^ punctuation.terminator.dart
```

Comment:
This is the important part - without the fix this code is considered a comment, but with this fix we're back to code.